src/main/groovy/com/blackducksoftware/integration/hub/detect/bomtool/CpanBomTool.groovy

```diff
@@ -58,8 +58,8 @@ class CpanBomTool extends BomTool {
     public boolean isBomToolApplicable() {
         def containsFiles = detectFileManager.containsAllFiles(sourcePath, 'Makefile.PL')
         if (containsFiles) {
-            cpanExecutablePath = executableManager.getPathOfExecutable(ExecutableType.CPAN, detectConfiguration.getCpanPath())
-            cpanmExecutablePath = executableManager.getPathOfExecutable(ExecutableType.CPANM, detectConfiguration.getCpanmPath())
+            cpanExecutablePath = getExecutablePath(ExecutableType.CPAN, true, detectConfiguration.getCpanPath())
```

Comment:
Is this supposed to be `findExecutablePath(....)` not `getExecutablePath(...)` ?